modules/ssm-parameters/main.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  sops_yaml     = yamldecode(data.sops_file.source.raw)
+  secret_params = nonsensitive(local.sops_yaml[var.sops_source_key])
+
+  secret_params_normalized = {
+    for key, value in local.secret_params :
+    key => {
+      name        = key
+      value       = value
+      description = "SecureString param created from ssm-parameters component from SOPS source file: ${var.sops_source_file} from key at ${var.sops_source_key}"
+      overwrite   = true
```

Comment:
Wouldn't it be safer to set this to `false` ?

Reply:
@nitrocode the point of this module is that your always managing your secrets via `SOPS` and then terraform is just injecting them into parameter store so systems that read from parameter store can have access to them. With that in mind, our thought process is that we always want the sops file to be the declarative source of truth and then this component should reconcile any out-of-band changes to the parameter store value that aren't made to the sops file. 

Does that make sense on why it's `true` here?